requirements/requirements.txt

```diff
@@ -6,5 +6,7 @@ lmfit >= 1.0.1
 matplotlib >= 2.0
 mpmath >= 1.0.0
 numpy >= 1.18.1
+pandas >= 1.1.0
 scipy >= 1.2
 setuptools >= 41.2
+xarray  >= 0.16.0
```

Comment:
@StanczakDominik You could answer this better than me, do we need to have version dependencies on `pandas` and/or `xarray`?  My intuition says to have no version dependency for `pandas` but have one for `xarray`.  Then, `xarray` will always grab the `pandas` version it needs.  For the `xarray` version, I'd grab the one where Python `>= 3.6` was first supported, so `xarray >= 0.14.0`.

Reply:
No idea, honestly! Pandas 1.0.0 has been out for a while, doesn't sound like a bad idea to have that as a lower bound; but I suspect xarray depends on it pandas, so we might as well go with their minimum dependency. For xarray, no idea.

Whatever we do with, the ultimate condition is that it works; so the minimum version should be added to the minimum- or minimal- or something environment in tox.ini, so that they're tested.

Sorry for lack of precision, just had a very efficient waking up and am a bit unfocused :sweat_smile:

Reply:
That makes sense - to do that I assume I just remove the >= #.#.# from the `setup.cfg` file. Do I delete the pandas line altogether from `requirements.txt` or just delete the version requirement?

Reply:
> No idea, honestly! Pandas 1.0.0 has been out for a while, doesn't sound like a bad idea to have that as a lower bound; but I suspect xarray depends on it pandas, so we might as well go with their minimum dependency. For xarray, no idea.

`xarray` does have a dependency on `pandas`.  I believe it's the main structure they use for indexing their `DataArray`s and `Dataset`s.

> That makes sense - to do that I assume I just remove the >= #.#.# from the setup.cfg file. Do I delete the pandas line altogether from requirements.txt or just delete the version requirement?

I prefer that we leave `pandas` in the `requirements.txt` and `setup.cfg` so it's explicitly clear that `plasmapy` has a dependency on `pandas` (only because we specifically import `pandas`).  I say we do `xarray >= 0.14.0` and `pandas >= 1.0.0` because they are the first  version to roll their dependencies to `python >= 3.6`.

Reply:
Ok, done (I think)